packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -260,15 +260,15 @@ contract Deploy is Deployer {
         }
     }
 
-    /// @notice Transfer ownership of a Proxy to the ProxyAdmin contract
+    /// @notice Transfer admin of a Proxy to the ProxyAdmin contract
     ///         This is expected to be used in conjusting with deployERC1967ProxyWithOwner after setup actions
     ///         have been performed on the proxy.
-    /// @param _name The name of the proxy to transfer ownership of.
+    /// @param _name The name of the proxy to transfer admin of.
```

Comment:
Any reason why not to also change owner of `InitializeDataAvailabilityChallenge` ?

Reply:
iirc, this function actually changed the admin - and not owner (as described in the comment) - and was updated for the sake of clarity.